src/components/todo/todo-connected.js

```diff
@@ -62,31 +67,80 @@ const ToDo = () => {
       .catch(console.error);
   };
 
+  const _deleteItem = () => {
+    fetch(todoAPI, {
+      method: 'delete',
+      mode: 'cors',
+    })
+    .then(response => response.json())
+  };
+
   useEffect(_getTodoItems, []);
 
+  useEffect(() => {
+    setCount(list.filter(item => !item.complete).length);
+   }, [list]);
+ 
+   useEffect(() => {
+     document.title = `To Do List: (${count})`;
+     }, [count]);
+ 
+ 
   return (
     <>
-      <header>
-        <h2>
-          There are {list.filter(item => !item.complete).length} Items To Complete
-        </h2>
-      </header>
-
-      <section className="todo">
-
-        <div>
-          <TodoForm handleSubmit={_addItem} />
-        </div>
-
-        <div>
-          <TodoList
-            list={list}
-            handleComplete={_toggleComplete}
-          />
-        </div>
-      </section>
+    <Router>
+      <Container>
+      <br />
+          <Navbar.Brand href="#home">Home</Navbar.Brand>
+          <Route path="/" exact component={_getTodoItems} />
+          <Route path="/edit/:id" component={_toggleComplete} />
+          <Route path="/create" component={_addItem} />
+          <Route path="/delete" component={_deleteItem} />
+        <Card>
+          <Card.Header bg="dark" > To Do List Manager ({count}) </Card.Header>
+          <Card.Body>
+            <section className="todo">
+              <div>
+                <TodoForm handleSubmit={_addItem} />
+              </div>
+                <div>
+                  <TodoList
+                    list={list}
+                    handleComplete={_toggleComplete}
+                  />
+                </div>
+            </section>
+          </Card.Body>
+        </Card>
+      </Container>
+    </Router>
     </>
   );
-};
+}
+
+
+  // return (
+  //   <>
+  //     <header>
+  //       <h2>
+  //         There are {list.filter(item => !item.complete).length} Items To Complete
+  //       </h2>
+  //     </header>
+
+  //     <section className="todo">
+
+  //       <div>
+  //         <TodoForm handleSubmit={_addItem} />
+  //       </div>
+
+  //       <div>
+  //         <TodoList
+  //           list={list}
+  //           handleComplete={_toggleComplete}
+  //         />
+  //       </div>
+  //     </section>
+  //   </>
+  // );
```

Comment:
I am not sure why you are using mongo, you shouldn't need it if you are using an API.